### PR TITLE
add a link to Ops handbook

### PR DESCRIPTION
**Pull request by kayiwa**
Alphabetized Links and removed "developer"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 The repository walks you through best practices, culture, and technological corner cases on PU Library. These are the parts that are tricky to fully automate or in some cases not possible to. These are also the shared norms and best practices agreed upon across teams in PUL IT which, because they are shared, do not fit solely into one team's handbook.
 
-Each developer team at PU Library also has its own team handbook:
+Each team at PU Library also has its own team handbook:
```
Comment (hackartisan) [inline README.md:5]: Suggestion: "The developer and operations teams also have their own team handbooks" since PU Library is quite a broad term :)

**Review by hackartisan — CHANGES_REQUESTED** [review]
Cool! Just a wording suggestion from me.

File: README.md (hunk 1 above)
Comment (acozine) [reply README.md]: Addressed in 51ad4112904e5b6ada96726a11d6241089453f35

**Review by acozine — APPROVED** [review]
Looks good, thanks for the comment @hackartisan! I put it in as a suggestion with attribution.

**Pull request merged**